src/reporter.cc

```diff
@@ -13,7 +13,6 @@
 // limitations under the License.
 
 #include "benchmark/reporter.h"
-#include "minimal_leastsq.h"
```

Comment:
please do. i don't want refactorings to benchmark_api to lead to missing symbols. everything should include what it uses.